pkg/nsx/services/common/compare.go

```diff
@@ -41,12 +41,15 @@ func CompareResources(existing []Comparable, expected []Comparable) (changed []C
 		if e2, ok := existingMap[key]; ok {
 			if isChanged := CompareResource(e2, e); !isChanged {
 				continue
+			} else {
+				log.V(1).Info("resource changed", "existing", e2, "expected", e)
```

Comment:
I guess `e2` is existing resource, `e` is expected one. The naming is confused..

Reply:
OK, updated.